packages/components/src/mobile/link-picker/link-picker-results.native.js

```diff
@@ -36,7 +37,7 @@ export default function LinkPickerResults( {
 
 	// a stable debounced function to fetch suggestions and append
 	const { fetchMoreSuggestions } = useSelect( ( select ) => {
-		const { getSettings } = select( 'core/block-editor' );
```

Comment:
This change is for the React Native file and it breaks the dependency chain because it depends on the packages that already depend on `@wordpress/components`. Let's leave the hardcoded string here for now.

This is the primary reason why this refactoring has started. Other benefits might get visible as the `@wordpress/data` API evolves 😄

Reply:
Gotcha :+1: 
Done